docs: add troubleshooting operator ns (#35005)
gravitational/teleport: docs/pages/management/dynamic-resources/teleport-operator.mdx

  To diagnose reconciliation issues, you will have to inspect all pods to find the one reconciling the resources.
</Admonition>

### The operator does not see the resource

> If you created the resource, can get it with `kubectl get <resource-type>`, but

Reviewer: ```suggestion
If you created a resource, and can get it with `kubectl get <resource-type>`, but
```

> the resource status stays empty and the operator does not produce logs, please

Reviewer: ```suggestion
the resource status stays empty and the operator does not produce logs,
```

check if the resource lives in the `teleport-cluster` namespace.

> The operator only watches for resource in its own namespace.

Reviewer: ```suggestion
The operator only watches for resources in its own namespace.
```
